components/builder-db/src/models/package.rs

```diff
@@ -461,30 +461,29 @@ impl Package {
             .limit(1)
             .get_result(conn);
 
-        let end_time = PreciseTime::now();
-        trace!("DBCall package::get_latest time: {} ms",
-               start_time.to(end_time).num_milliseconds());
-        Histogram::DbCallTime.set(start_time.to(end_time).num_milliseconds() as f64);
-        Histogram::GetLatestPackageCallTime.set(start_time.to(end_time).num_milliseconds() as f64);
+        let duration_millis = start_time.elapsed().as_millis();
+        trace!("DBCall package::get_latest time: {} ms", duration_millis);
+        Histogram::DbCallTime.set(duration_millis as f64);
+        Histogram::GetLatestPackageCallTime.set(duration_millis as f64);
 
         result
     }
 
     pub fn get_all_latest(conn: &PgConnection) -> QueryResult<Vec<PackageWithVersionArray>> {
         Counter::DBCall.increment();
-        let start_time = PreciseTime::now();
+        let start_time = Instant::now();
         let result = origin_packages_with_version_array::table
             .distinct_on((origin_packages_with_version_array::origin, origin_packages_with_version_array::name, origin_packages_with_version_array::target))
             .order(sql::<PackageWithVersionArray>(
                 "origin, name, target, string_to_array(version_array[1],'.')::\
                 numeric[] desc, ident_array[4] desc",
             ))
             .get_results(conn);
-        let end_time = PreciseTime::now();
+        let duration_millis = start_time.elapsed().as_millis();
```

Comment:
These variables have been inconsistently named  through the pr, some are `duration_milli` some are `duration_millis` which, is not functionally a concern but could make searching the code one query longer. Not a merge blocker but my personal preference would be to keep them consistently named.

Reply:
:man_facepalming: I'll fix that